static/js/stream_muting.js

```diff
@@ -21,7 +21,7 @@ export function update_is_muted(sub, value) {
             saved_ypos = message_viewport.scrollTop();
         } else if (
             message_lists.home === message_lists.current &&
-            message_lists.current.selected_row().offset() !== null
+            message_lists.current.selected_row().offset() !== (null || undefined)
```

Comment:
What's the story with this change? It seem unlike the others.

Reply:
I was getting errors on Chrome with this null check not working, throwing the error that message_lists.current.selected_row().offset() was undefined . Checking for undefined as well seemed to work. I also don't get this behavior on chat.zulip.org , only in development so its possible that it's just an error specific to my machine. My bad for having this in this pull request, I'll remove it if I do end up getting this ready for review